template.go

```diff
@@ -301,10 +303,20 @@ const (
 			}
 		{{end}}
 
+		// SkipMinimockFinish makes minimock to skip methods call final check, making it to work in
+		// '0 or more' calls for all mocked methods. Without it, all methods mocked with 'Return()' are
+		// obligated to be called at least once ('1 or more' requirement) which is not a convenient
+		// restriction in some test scenarios.
+		// It is NOT RECOMMENDED to use this option by default unless you really need it, as it helps to
+		// catch the problems when the expected method call is totally skipped during test run.
+		func (m *{{$mock}}{{(paramsRef)}}) SkipMinimockFinish() {
+			m.skipMinimockFinishCheck = true
+		}
```

Comment:
It would be nice to set this property for each individual method of an interface rather than set this for all methods of the mock to have some flexibility.

I'm thinking about something like:
```go

m := minimock.NewController()

storageMock := NewStorageMock(m).
   StoreBoxMock.Optional().Expect(box).Return(nil).
   StoreMail().Expect(envelope).Return(nil)
```